style.css

```diff
@@ -0,0 +1,552 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+.header__social__media,
+.un__icon,
+.nav__items,
+.sponsors,
+.footer {
+  display: none;
+}
+
+.see__program {
+  display: none;
+}
+
+.hamburger.active span:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active span:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active span:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+.mobile__nav__menu {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  position: fixed;
+  left: -100%;
+  top: 30px;
+  width: 100%;
+  height: 100%;
+  transition: 0.3s;
+  background-color: rgba(0, 0, 0, 0.3);
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+}
+
+.mobile__nav__menu.active {
+  left: 0;
+}
+
+.mobile__nav__items {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  margin-left: 20px;
+  gap: 25px;
+  top: 50px;
+  height: 100vh;
+}
+
+.mobile__nav__links {
+  font-weight: 600;
+  color: #fff;
+  font-size: 30px;
+}
+
+.bar {
+  display: block;
+  height: 3px;
+  width: 25px;
+  margin: 5px;
+  background-color: #272a31;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+}
+
+.home {
+  display: flex;
+  flex-direction: column;
+  background-image: url("./images/bkgrnd.png");
+  background-position: top center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 50px;
+  margin-bottom: 30px;
+}
+
+.home__title {
+  font-weight: 300;
+  font-size: 1.5rem;
+  color: #ec5242;
+}
+
+.home__subtitle {
+  font-size: 50px;
+  font-weight: 700;
+  background: url(./images/font-color.jpg);
+  background-clip: content-box;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.home__description {
+  background-color: #d3d3d3;
+  border: 4px solid white;
+  color: #272a31;
+  padding: 1%;
+  width: 60%;
+  margin-top: 20px;
+  margin-right: 100px;
+}
```

Comment:
- [OPTIONAL] KIndly make sure to increase the width of this paragraph in order to match the provided Behance design on the mobile version.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/225732125-b6554830-3f0a-4f63-af1b-49b19a9970fc.png)

   - ### BEHANCE DESIGN
   
   ![image](https://user-images.githubusercontent.com/87197412/225732213-dcdeab3a-99c3-40f5-8144-4b955bb73a58.png)

Reply:
Highly noted, will do thank you.